programmers/비밀지도.js

```diff
@@ -0,0 +1,32 @@
+function solution(n, arr1, arr2) {
+    const passwords = [];
+    const results = [];
+    
+    let sum = 0;
+    let str = '';
+
+    for (let i = 0; i < arr1.length; i++) {
+        sum = arr1[i] | arr2[i];
```

Comment:
```suggestion
        sum = arr1[i] | arr2[i];
```
비트연산자는 생각 못했는데 하나 배워가요 :)